data/stat-effects.ts

```diff
@@ -0,0 +1,149 @@
+export const statEffects = {
+  mobility: {
+    units: '%',
+    values: {
```

Comment:
Any feeling on just making this an array instead of a string-keyed object where all the keys are strings of consecutive numbers?

Reply:
I was torn when I did it, I largely did it this way as it was easier to cross check that values were right from the game and crowd sources. In saying that I have no problem in converting it over.

Reply:
Fine either way, it just seemed odd